### DISCUSSION TITLE
How Max Text Length for Invoices and Quotes extend over 300 chars

### DISCUSSION THREAD

**Discussion by mad99** [YetiForceCompany/YetiForceCRM#14988] [How to] [answered]
HI,

**- Has anybody a hint where I cant setup a different max_legth for invoices-description Text for each product-item?**

Currently the description is cut after about 300 characters.

Was looking in Google, Git, YF-Settings, Default-Module Settings, Extended Module-Settings found something in «Edit fields» and there I have tried with «infoices» and «products» but was no change in the PDF-View. Still after 300 char cut and follwing [...] three dots.

Guess I'm looking for the wrong term. Would be great to get help or push in the right direction.

**Comment by mariuszkrzaczkowski** [comment]
give a screenshot

**Comment by rskrzypczak** [comment, marked as answer]
You have two options:
1. Disable text truncation by adding it to the config/Modules/FInvoice.php file 
```
namespace Config\Modules;

/**
 * Configuration file: Config\Modules\FInvoice.
 */
class FInvoice
{
	public static $inventoryCommentIframeContent = false;
```
2. Change the length directly in the file
    https://github.com/YetiForceCompany/YetiForceCRM/blob/d900ebe3f9eed9779add3d559f41bc1342f91cde/modules/Vtiger/inventoryfields/Comment.php#L79

**Reply by mad99** [reply]
Hey @rskrzypczak and @mariuszkrzaczkowski 
Thank you for your help.

your second approach @rskrzypczak is what I'm looking for. 
I set it to 6000 ;)
Works perfect - Thank you!

____
While the first solution with the 	`public static $inventoryCommentIframeContent = false;`
deleted all the description-text in the invoice. I tried also with `true` or with any INT like `600`.

This seams not to work like mentioned. It always cut ALL or at least has no change in the length of the description. The `...` are still exactely at the same point in value.
